package.json

```diff
@@ -32,21 +35,13 @@
     "truffle-hdwallet-provider": "0.0.3"
   },
   "devDependencies": {
-    "eslint-plugin-import": "2.7.0",
+    "eslint": "4.13.1",
+    "eslint-config-airbnb": "16.1.0",
+    "eslint-plugin-import": "2.8.0",
+    "eslint-plugin-jsx-a11y": "6.0.3",
```

Comment:
Do we use jsx anywhere in this repo?

Reply:
Not sure. I think that one just comes with eslint by default.

Reply:
For some reason, we need it. Otherwise: 

```ESLint: 4.13.1.
ESLint couldn't find the plugin "eslint-plugin-jsx-a11y". This can happen for a couple different reasons:

...

2. If ESLint is installed locally, then it's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm i eslint-plugin-jsx-a11y@latest --save-dev

If you still can't figure out the problem, please stop by https://gitter.im/eslint/eslint to chat with the team.```